### DISCUSSION TITLE
Wizardry 6 sound help

### DISCUSSION THREAD

**Discussion by bsmith1702** [dosbox-staging/dosbox-staging#4050] [Q&A] [answered]
Hi,
     I am trying to get Wizardry 6 working.  I changed the sound settings in the conf file to what was recommended, but I don't get any sound.  The in-game configuration menu just has options to turn the sound on or off and the GOG version does not have an installer, so I don't think there is a game setting that I can change.  I feel like I am missing something.  I've attached the conf file (changed the file name to txt so I could post it).

Thanks

[bane.conf.txt](https://github.com/user-attachments/files/17689404/bane.conf.txt)

**Comment by johnnovak** [comment, marked as answer]
You have the PC speaker disabled in your config. The game uses the PC speaker for all sounds, and the version I tried also allows to select AdLib in the config utility. But the PC speaker option is the best for this game anyway; don't bother with AdLib.

So setting `pcspeaker = impulse` should be all you need. I also recommend setting `reverb = tiny`.

For reference, I have a `WINSTALL.EXE` in the game folder that lets me change the configuration.

Maybe go complain to GOG to stop mutilating their games and include all the original files 🤷🏻 Or just stop buying DOS games from them (it's a bit pointless, really), and start using eXoDOS 😎 🤘🏻 

<img width="1215" alt="image" src="https://github.com/user-attachments/assets/ca10ba48-888e-4830-ade7-20bfebdcb9fc">

**Reply by bsmith1702** [reply]
Okay.  Thank you.  It works now.

I was following the instructions in the release notes which said to set `pcspeaker = off`, but I guess that was for using Adlib sound.

**Reply by johnnovak** [reply]
> Okay. Thank you. It works now.
> 
> I was following the instructions in the release notes which said to set `pcspeaker = off`, but I guess that was for using Adlib sound.

Indeed, and no need to guess because I said so in the notes 😄 
https://www.dosbox-staging.org/releases/release-notes/0.82.0/#adlibopl-dc-bias-remover
